packages/wdio/src/wdio.ts

```diff
@@ -33,16 +33,17 @@ export async function getAxeVersion(driver: WdioBrowser): Promise<typeof axeVers
 }
 
 /**
- * Load axe source into browser if it is not already loaded and return version of axe
+ * Load axe source into browser if it is not already loaded and return version of axe.
+ * Keep waiting and trying until given timeout milliseconds.
  */
-export async function loadAxe(driver: WdioBrowser): Promise<void> {
-    if ((await getAxeVersion(driver)) !== axeVersion) {
+export async function loadAxe(driver: WdioBrowser, timeout = 1000, pollTime = 100): Promise<void> {
```

Comment:
Is one second a long enough default timeout? This function executes Javascript in the browser. Feels like that time could vary greatly depending on the target browser.

Reply:
Agree, increased the timeout and changed the polling logic to load axe only once.